src/main/java/com/google/devtools/build/lib/syntax/StarlarkSemantics.java

```diff
@@ -164,6 +166,8 @@ public boolean flagValue(FlagIdentifier flagIdentifier) {
 
   public abstract boolean incompatibleDisallowStructProviderSyntax();
 
+  public abstract boolean incompatibleDisallowRuleExecutionPlatformConstraintsAllowed();
```

Comment:
I know this is nitty, but the docs above mandate listing variables here and in the other places alphabetically, and I know there's some precedent for enforcing that.

Reply:
Good point, I'll fix that on the imported version.